starters/cms/src/FieldMapping.tsx

```diff
@@ -212,14 +223,16 @@ export function FieldMapping({ collection, dataSource, initialSlugFieldId }: Fie
                 </div>
 
                 <footer>
-                    <hr className="sticky-top" />
-                    <button disabled={isSyncing} tabIndex={0}>
+                    <hr />
+                    <button disabled={isSyncing || !isAllowedToManage} tabIndex={0}>
                         {isSyncing ? (
                             <div className="framer-spinner" />
-                        ) : (
+                        ) : isAllowedToManage ? (
                             <span>
                                 Import <span style={{ textTransform: "capitalize" }}>{dataSourceName}</span>
                             </span>
+                        ) : (
+                            "Insufficient permissions"
```

Comment:
In the Framer app when you don't have sufficient permission we show exactly the same UI, the only difference is that it's disabled. Do you think without changing the text here it's a confusing experience?

Reply:
Disabled submit buttons traditionally mean "something is missing in your input", overloading that is a no-go, IMO. I considered adding a pill above the disabled button, explaining what's wrong, but chose not to as to not get bogged down by now having to design a whole new thing 😄 I completely agree that ideally buttons shouldn't change their text. I also agree that we should inspire people with good UI out of the box, so will look into improving this!

Reply:
Okay. Let's then not use a form but make it a regular disabled button

Reply:
Or just change the type of the button to button in case its disabled

Reply:
Oh, but it's still semantically a submit button. Didn't mean it in a "let's be pedantic about HTML spec" way 😄